tox.ini

```diff
@@ -124,7 +124,8 @@ allowlist_externals = sh
 setenv =
   ; override CONTRIB_REPO_SHA via env variable when testing other branches/commits than main
   ; i.e: CONTRIB_REPO_SHA=dde62cebffe519c35875af6d06fae053b3be65ec tox -e <env to test>
-  CONTRIB_REPO_SHA={env:CONTRIB_REPO_SHA:main}
+  ; CONTRIB_REPO_SHA={env:CONTRIB_REPO_SHA:main}
+  CONTRIB_REPO_SHA=e36889568d1c57c5f6e1dfa65a73519a8eb6607d
```

Comment:
Is this intentional?

Reply:
yeah, for the time being I need this to be fixed to a particular commit until https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2687 gets merged